app/services/notifications/setup/update-notifications.service.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+/**
+ * Update notifications
+ * @module UpdateNotificationsService
+ */
+
+const ScheduledNotificationModel = require('../../../../app/models/scheduled-notification.model.js')
+
+/**
+ * Update notifications
+ *
+ * This query uses postgresql's ability to insert with batches. There is a limitation on the batch size
+ * https://www.postgresql.org/docs/current/postgres-fdw.html#POSTGRES-FDW-OPTIONS-REMOTE-EXECUTION. The calling
+ * service will handle the batch process.
+ *
+ * @param {object} notifications
+ *
+ * @returns {object} - the Updated notifications
+ */
+async function go(notifications) {
+  return ScheduledNotificationModel.query().insert(notifications).onConflict('id').merge(['status', 'notifyStatus'])
```

Comment:
This is just a question.

Will we be using the scheduled notifications returned from this service somewhere else? If not, this should just be `await ScheduledNotificationModel`.

Reply:
correct it should be await i will update